README.md

```diff
@@ -206,8 +206,9 @@ Services to securely store your Docker images.
 * [GCE Container Registry](https://cloud.google.com/container-registry/) Fast, private Docker image storage on Google Cloud Platform
 * [GitLab Container Registry](https://docs.gitlab.com/ce/container_registry/README.html) - Repositories focused on using it images in GitLab CI
 * [Rescoyl](https://github.com/noteed/rescoyl) - Private Docker registry (free and open source) by [@noteed][noteed]
+* [Sonatype Nexus](https://www.sonatype.com/nexus-repository-oss) - Repository with Universal Support, also for Docker images
 * [Quay.io](https://quay.io/) (part of CoreOS) - Secure hosting for private Docker repositories
```

Comment:
Hi @gtonic Thanks for the submission. Could you please move Quay above Rescoy so we alphabetize this section a bit better?